scripts/_settings.sh

```diff
@@ -40,7 +41,7 @@ fi
 if [ ! -d "$SCRIPTDIR/vendor/bash_ini_parser" ]; then
   echo ""
   echo "Installing Bash ini parser..."
-  git clone https://github.com/rudimeier/bash_ini_parser.git "$SCRIPTDIR/vendor/bash_ini_parser"
+  git clone https://github.com/pmgouveia/bash_ini_parser.git "$SCRIPTDIR/vendor/bash_ini_parser"
```

Comment:
hm that referral only works if u pay 5$ its a pyramid scheme! :o anyway I have digital ocean, i will spin a droplet